skins/cat17/faq-page/src/Faq.vue

```diff
@@ -12,14 +12,16 @@
  	<h2>{{ messages.page_title }}</h2>
  	<h5>{{ messages.page_subtitle }}</h5>
  	<div class="row">
- 	<div class="col-xs-12 col-sm-9">
+ 	<div class="col-xs-12 col-sm-8">
 	  	<search-bar :messageSearch="messages.search"></search-bar>
 		<div class="form-shadow-wrap">
-			<h2 v-if="!isPreview" class="title space-below underlined">{{ topicTitle }}</h2>
+			<h2 v-if="!isPreview" class="title">{{ topicTitle }}</h2>
```

Comment:
I remember talking about this - "Preview" means "Overview", right? Could you rename this to make it more clear?